### PR TITLE
feat(api): template bulk update parallel

### PR DESCRIPTION
**Pull request by richardlt**
Signed-off-by: richardlt <richard.le.terrier@gmail.com>

1. Description
1. Related issues
1. About tests
1. Mentions

@ovh/cds

### CODE REVIEW

**Review by yesnault — APPROVED** [review]

File: cli/cdsctl/template_bulk.go [hunk 1]
```diff
@@ -53,6 +53,11 @@ var templateBulkCmd = cli.Command{
 			Name:  "track",
 			Usage: "Wait the bulk to be over",
 		},
+		{
+			Type:  cli.FlagBool,
+			Name:  "parallel",
+			Usage: "Apply template on workflow in parallel",
+		},
 	},
 }
```
Comment (fsamin) [inline cli/cdsctl/template_bulk.go:63]: is there now a "non interactive" mode for this command ?

File: engine/api/workflow_template_bulk.go [hunk 2]
```diff
@@ -0,0 +1,337 @@
...
+	ticker := time.NewTicker(tick)
+	defer ticker.Stop()
+	for {
+		select {
+		case <-ctx.Done():
+			return ctx.Err()
+		case <-ticker.C:
+			bs, err := workflowtemplate.GetBulksPending(ctx, api.mustDBWithCtx(ctx))
+			if err != nil {
+				log.Error(ctx, "WorkflowTemplateBulk> unable to start tx: %v", err)
```
Comment (fsamin) [inline engine/api/workflow_template_bulk.go:33]: not sure it's the right error message

**Review by fsamin — CHANGES_REQUESTED** [review]

File: cli/cdsctl/template_bulk.go (hunk 1 above)
Comment (richardlt) [reply cli/cdsctl/template_bulk.go:63]: yes and output was fixed in a previous PR

**Review by fsamin — APPROVED** [review]

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#18317.0 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✘  
  * Test SDK ✔ 
  * Test UI ✔

**Pull request merged**